tests/modules/bamtools/split/test.yml

```diff
@@ -1,10 +1,23 @@
-- name: bamtools split test_bamtools_split
-  command: nextflow run ./tests/modules/bamtools/split -entry test_bamtools_split -c ./tests/config/nextflow.config -c ./tests/modules/bamtools/split/nextflow.config
+- name: bamtools split test_bamtools_split_single_input
+  command: nextflow run tests/modules/bamtools/split -entry test_bamtools_split_single_input -c tests/config/nextflow.config
```

Comment:
```suggestion
  command: nextflow run ./tests/modules/bamtools/split -entry test_bamtools_split_single_input -c ./tests/config/nextflow.config -c ./tests/modules/bamtools/split/nextflow.config
```

Reply:
This is the syntax we use in all other modules so we should keep it the same. Helps when programmatically updating all modules at the same time if required.

Reply:
That's weird, because this file was generated by `nf-core`